interview-questions.md

```diff
@@ -8,43 +8,67 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. As a developer, I am creating a Rails application with a model called Cohort that has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
   Your answer:
+  It is my understanding you would have to create a new column and name it the same as the foreign key that you want to pass it.
 
   Researched answer:
-
+It is my understanding you would have to create a new column and name it the same as the foreign key that you want to pass it. You would also need to use the convention of has_many__?__.  Once you updated the model Cohort  you would also need to update the foreign key of the Student model.
```

Comment:
Correct, the foreign key is on the Student model.  You certainly would want to check to make sure that both models are set up with the proper associations.  To update the Student model to have the foreign key, you would want to run a migration to update the schema to have the added column.